util/index.js

```diff
@@ -8,3 +10,33 @@ export function preprocessJsonInput(text) {
   }
 }
 export {parseSite} from './page-parser.js';
+
+export function createTestFile(filePath) {
+  const boilerPlate = `import { test, expect } from '@playwright/test';
+
+test('generated test', async ({ page }) => {
+`;
+
+  fs.writeFileSync(filePath, boilerPlate.trim() + '\n');
+}
+
+export function appendToTestFile(userInput, generatedCode, filePath) {
+  const indentation = 1;
+  let formattedCode = `\t// ${userInput}\n`;
+
+  // Split the code into lines and format each line
+  const lines = generatedCode.split('\n');
+  for (const line of lines) {
+    formattedCode += formatCodeLine(line, indentation);
+  }
+
+  fs.appendFileSync(filePath, formattedCode, 'utf8');
+}
+
+export function completeTestFile(filePath) {
+  fs.appendFileSync(filePath, '});', 'utf8');
+}
+
+function formatCodeLine(codeLine, indentation) {
+  return `${'\t'.repeat(indentation)}${codeLine}\n`;
+}
```

Comment:
then this is no longer needed

Reply:
https://github.com/mayt/BrowserGPT/pull/16/commits/c0f6332366e6caae89b96da31ffc0cc122a7ddd1

Thanks! I think I initially had used spaces instead of tab so it made sense lol

Reply:
https://github.com/mayt/BrowserGPT/pull/16/commits/05e8634dbec02d45d575c2f13dd527424ecaaf8b